docs/reference/sailing/turning.md

```diff
@@ -1,9 +1,22 @@
 # Types of Turns
 
-In sailing, there are 4 distinct types of turns. Read the descriptions below, and observe how they fit into the diagram.
+In sailing, there are 4 distinct types of turns. Read the descriptions below, and observe how they fit into the diagrams.
 
 Note that any of these types of turn can be done in either the clockwise or counter-clockwise directions.
 
+## Classifying Types Of Turns Summary
+
```

Comment:
Could put a sentence here to describe the flowchart

Reply:
I added this. I was kinda hoping the title would speak for itself haha but I don't mind adding a sentence.